docs/fields-gen.asciidoc

```diff
@@ -1,23 +1,28 @@
 [[ecs-base]]
 === Base fields
 
-The base set contains all fields which are on the top level. These fields are common across all types of events.
+The base set contains top level fields that are common across all types of events
```

Comment:
We should always quote field names -- such as `base` above -- in backticks, to get the monospace visuals. Otherwise this can make for strange-sounding sentences.

Reply:
Noted. This particular example is populated by generated content, so we'll need make those adjustments in the schema files.

Reply:
Haha oh jeez. You can see I'm not used to reviewing the asciidoc yet 🤦‍♂️ 

I'll take care of this one :-)